operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -624,6 +634,73 @@ private void handleDeletedDomain(Domain domain) {
         .execute();
   }
 
+  /**
+   * Dispatch the Cluster event to the appropriate handler.
+   *
+   * @param item An item received from a Watch response.
+   */
+  public void dispatchClusterWatch(Watch.Response<Cluster> item) {
+    switch (item.type) {
+      case "ADDED":
+        handleAddedCluster(item.object);
+        break;
+      case "MODIFIED":
+        handleModifiedCluster(item.object);
+        break;
+      case "DELETED":
+        handleDeletedCluster(item.object);
+        break;
+
+      case "ERROR":
+      default:
+    }
+  }
+
+  private void handleAddedCluster(Cluster cluster) {
+    System.out.println("DomainProcessorImp.handleAddedCluster cluster: " + cluster);
+    DomainPresenceInfo cachedInfo = getExistingDomainPresenceInfo(cluster.getNamespace(), cluster.getDomainUid());
+    System.out.println("DomainProcessorImp.handleAddedCluster cachedInfo: " + cachedInfo);
+    if (cachedInfo == null) {
+      return;
+    }
+
+    //LOGGER.info(MessageKeys.WATCH_DOMAIN, domain.getDomainUid());
```

Comment:
Find and delete commented out code.

Reply:
Oh, wait... This seems like code that should be uncommented and should initiate domain processing.

Reply:
Sorry, I haven't cleaned up as I'm currently working on refactoring and abstracting out the conditions for initiating domain processing for either Domain and/or Cluster events.